bin/git-fork

```diff
@@ -16,6 +16,7 @@ read user
 # extract owner + project from repo url
 project=${1##*/}
 owner=${1%/$project}
+[[ $project == *.git ]] && project=${project%.git}
```

Comment:
You don't need to do the test here. Just `project=${project%.git}` is fine.
If the suffix is there, it will be removed; if not, nothing happens.